fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/consensus/LearnValue.java

```diff
@@ -0,0 +1,42 @@
+package com.github.dedis.popstellar.model.network.method.message.data.consensus;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+public final class LearnValue {
+
+  @SerializedName("decision")
```

Comment:
Is this necessary ?

Reply:
No, the @SerializedName is redundant because the variable name is the same.
But keeping it could maybe avoid problems if someone in a few years try to refactor some of theses classes.